feat(engine): explain "Unterminated string literal" error (#2)
mattpocock/ts-error-translator: packages/engine/errors/1002.md

---
original: "Unterminated string literal."

> excerpt: "You've an unterminated string literal somewhere. String literals must be enclosed by single (') or double (\") quotes."

Reviewer: ```suggestion
excerpt: "You've started a string (via a single or double quote) but haven't ended it."
```

---

To fix this error, check if: 

- you have opening and closing quotes (single or double) for your string literal
- you have escaped your string literal correctly (by prefixing it with `/`)
- your string literal isn't split across multiple lines

> ## Examples

Reviewer: ```suggestion
### Examples
```
